packages/server/assets/workspacesCore/typedefs/workspaces.graphql

```diff
@@ -145,7 +145,11 @@ type Workspace {
   Active user's role for this workspace. `null` if request is not authenticated, or the workspace is not explicitly shared with you.
   """
   role: String
-  team(filter: WorkspaceTeamFilter): [WorkspaceCollaborator!]!
+  team(
```

Comment:
This will likely break Mike's members frontend, coordinate with him on how to release this. Maybe his adjustments to the FE should be done in this same branch to avoid deploying a BE change that breaks the FE.